website/pages/docs/glossary/soc2.md

```diff
@@ -0,0 +1,42 @@
+2. Availability: This category measures the reliability and accessibility of a company's systems and infrastructure to ensure that they are available for use when needed.
+3. Processing integrity: This category evaluates the accuracy, completeness, and timeliness of a company's processing operations, including data entry, processing, and output.
+4. Confidentiality: This category assesses the company's policies and procedures for protecting sensitive data from unauthorized access, disclosure, and misuse.
+5. Privacy: This category evaluates the company's practices for collecting, using, and disclosing personal information in accordance with applicable laws and regulations.
+
+# Why Does SOC2 Compliance Matter?
+
+SOC2 compliance is important for several reasons:
+
+1. Protecting customer data: SOC2 compliance helps ensure that companies are implementing adequate safeguards to protect their customers' data from unauthorized access, theft, and other security incidents.
+2. Demonstrating trustworthiness: SOC2 compliance demonstrates that a company takes data security and privacy seriously, which can help build trust with customers, partners, and investors.
+3. Meeting regulatory requirements: SOC2 compliance may be required by certain industries or regulatory bodies, such as HIPAA for healthcare organizations.
+4. Competitive advantage: SOC2 compliance can give a company a competitive advantage by demonstrating a commitment to data security and privacy that sets it apart from its competitors.
+
+# How to Achieve SOC2 Compliance?
+
+Achieving SOC2 compliance requires a comprehensive approach to data security and privacy. Here are the key steps to achieving SOC2 compliance:
+
+1. Define scope: Identify the systems, processes, and data that are in scope for SOC2 compliance.
+2. Conduct a risk assessment: Assess the risks to the confidentiality, integrity, and availability of the systems, processes, and data in scope.
+3. Implement controls: Implement controls to mitigate the identified risks, such as access controls, network security, and data encryption.
+4. Monitor and test: Monitor and test the effectiveness of the controls to ensure ongoing compliance.
+5. Obtain an independent audit: Engage an independent auditor to conduct an SOC2 audit and issue a report.
+
+# Open Source SOC2 Report
+
+Previously we covered [what is the modern data stack](https://www.cloudquery.io/blog/what-is-the-modern-data-stack) and how we can build an infrastructure data lake. By having an infrastructure data lake you can easily build your own customizable CIEM with just standard SQL queries and views that you can monitor and visualize with your go-to BI tools and avoid the yet-another-dashboard fatigue and learning new proprietary query languages. Check out our full guide on how to use CloudQuery, pre-built queries, views and Grafana dashboards to build an open-source SOC2 Report (a TODO blog)
```

Comment:
```suggestion
Previously we covered [what is the modern data stack](/blog/what-is-the-modern-data-stack) and how we can build an infrastructure data lake. By having an infrastructure data lake you can easily build your own customizable CIEM with just standard SQL queries and views that you can monitor and visualize with your go-to BI tools and avoid the yet-another-dashboard fatigue and learning new proprietary query languages. Check out our full guide on how to use CloudQuery, pre-built queries, views and Grafana dashboards to build an open-source SOC2 Report (a TODO blog)
```

Reply:
Also not sure what `(a TODO blog)` means in this context, or which full guide we're referring to here